### DISCUSSION TITLE
Extra repository for examples

### DISCUSSION THREAD

**Discussion by bonh** [MeasureTransport/MParT#134] [Ideas]
@mparno suggested in #117 to "create a new repository [...] to hold the examples". Called "maybe something like `MParT-examples`". 

I think this is a good idea, including the name. This would involve to

1. create a new repo, 
2. move the folder [examples](https://github.com/MeasureTransport/MParT/tree/main/examples) into it,
3. construct the binder.
4. Add a reference to MParT-examples in the README and documentation of MParT.

I would prefer if we would keep the git history while moving the examples into the new repo.

What do you think?

**Comment by rubiop** [comment]
I think that's great, not sure what you mean by keep the git history ?

**Reply by bonh** [reply]
Every file has specific commits which reference that file. If we just copy the examples into a new repo this information and metadata are lost. If we keep the git history the specific commits still reference the example files but now it is a new repo.

**Reply by bonh** [reply]
Mhh, that explanation is not as smooth as it was in my mind...

**Reply by rubiop** [reply]
no, no I understand thanks! That's sounds cleaner indeed!
